content/departments/product-engineering/engineering/cloud/delivery/managed/enable_executors_process.md

```diff
@@ -0,0 +1,87 @@
+# Enabling executors
+
+Enabling executor on a managed instances is initiated by "?". There are a few logical steps to follow.
+
+1. Update the managed instances terraform to include executor modules
+1. Setup billing alert for the executor resources
+
+## Update the managed instances terraform
+
+The change to terraform will be done by Batch Change team. Batch Change teamm will ping us the Pull Request for the changes and Delivery team will apply the change.
+
+## Setup billing alert
+
+As of 2022-03-08, we give customers $500 compute credits for executors usage, so we need to setup billing alert.
+
+At a high level, we will create a new monitoring alerting channel (via email) in the GCP project, and configure a budget alert in our organization billing account. All billing alerts will be delivered to [#wg-shipping-executors](https://sourcegraph.slack.com/archives/C02MR5PPMKJ).
+
+### Create slack email integration
```

Comment:
Both of these were created by individual users, so they're vulnerable to breaking if those accounts get shut down. We should create a followup to replace these with something managed by a service account/bot.

Reply:
Based on my past experience with `Email` integration, when the creator's slack account gets deactivated (i.e. they leave Sourcegraph), we won't be able to make changes anymore, but the integration will stay active.

But yeah, we need something more reliable that is tied to an individual.

Reply:
this is tracked here https://github.com/sourcegraph/customer/issues/748